motion_systemvalidation/sub-pp002/sub-pp002_scans.tsv

```diff
@@ -1,5 +1,5 @@
 filename	acq_time
-motion/sub-pp002_task-backwards_tracksys-imu_motion.tsv	2023-02-17T14:06:53.308Z
-motion/sub-pp002_task-backwards_tracksys-omc_motion.tsv	2023-02-17T14:06:53.308Z
-motion/sub-pp002_task-obstacleHigh_tracksys-imu_motion.tsv	2023-02-17T14:06:57.171Z
-motion/sub-pp002_task-obstacleHigh_tracksys-omc_motion.tsv	2023-02-17T14:06:57.171Z
+motion/sub-pp002_task-backwards_tracksys-imu_motion.tsv	2023-05-05T17:39:47.307Z
+motion/sub-pp002_task-backwards_tracksys-omc_motion.tsv	2023-05-05T17:39:47.307Z
+motion/sub-pp002_task-obstacleHigh_tracksys-imu_motion.tsv	2023-05-05T17:39:51.873Z
+motion/sub-pp002_task-obstacleHigh_tracksys-omc_motion.tsv	2023-05-05T17:39:51.873Z
```

Comment:
Is this potentially PII? BIDS generally recommends date anonymization. See [Units](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#units):

> Dates can be shifted by a random number of days for privacy protection reasons. To distinguish real dates from shifted dates, is is RECOMMENDED to set shifted dates to the year 1925 or earlier. Note that some data formats do not support arbitrary recording dates. For example, the [EDF](https://www.edfplus.info/) data format can only contain recording dates after 1985. For longitudinal studies dates MUST be shifted by the same number of days within each subject to maintain the interval information. For example: `1867-06-15T13:45:30`

Reply:
I used the date when I converted the dataset to BIDS. That date is not related to anything.